onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -81,15 +81,15 @@ def from_string(t):
         except KeyError:
             raise ValueError()  # noqa: B904
 
-    @staticmethod
-    def to_proto(qtype):
-        if qtype == QuantType.QInt8:
+    @property
```

Comment:
```suggestion
```

perhaps making it a method is more suitable since there it is a “conversion process” rather than an inherent property of the object. But no strong opinion

Reply:
Especially when it will also raise in cases.

Reply:
Maybe a second opinion?